{"uid":"9079123045843dd9","name":"Add project to workspace test","fullName":"tests.workspaces_test.TestWorkspaces#test_add_project_to_workspace","historyId":"557b6733f5784cd469824aeac0e8ff06","time":{"start":1751594231408,"stop":1751594231408,"duration":0},"description":"Selecting and adding a project to workspace","descriptionHtml":"<p>Selecting and adding a project to workspace</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1751594178377,"stop":1751594178377,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1751594178378,"stop":1751594178378,"duration":0},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/conftest.py\", line 60, in data\n    with open(json_path, encoding=\"utf-8\") as json_file:\n         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"_verify_url","time":{"start":1751594178378,"stop":1751594178378,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"session_request","time":{"start":1751594178378,"stop":1751594178378,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Selecting and adding a project to workspace","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","steps":[{"name":"public ip address","time":{"start":1751594234226,"stop":1751594234235,"duration":9},"status":"passed","steps":[],"attachments":[{"uid":"5c302dfed2ffce9e","name":"HTTP logs of http://checkip.amazonaws.com/","source":"5c302dfed2ffce9e.txt","type":"text/plain","size":356}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"e30cb458e94ebdcf","name":"Cookies","source":"e30cb458e94ebdcf.json","type":"application/json","size":4117},{"uid":"3cba361e6950a6c4","name":"Session Storage","source":"3cba361e6950a6c4.json","type":"application/json","size":359},{"uid":"ce9c38b9ccc1f9b","name":"Local Storage","source":"ce9c38b9ccc1f9b.json","type":"application/json","size":548},{"uid":"f0580f3591452fe8","name":"Full Page Screenshot","source":"f0580f3591452fe8.png","type":"image/png","size":1822184},{"uid":"8f3d436d93178370","name":"URL","source":"8f3d436d93178370.uri","type":"text/uri-list","size":23},{"uid":"f779f395239d2274","name":"Network Logs","source":"f779f395239d2274.json","type":"application/json","size":7932312}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":7,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"session_request::0","time":{"start":1751594257573,"stop":1751594257573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"epic","value":"Workspaces"},{"name":"severity","value":"normal"},{"name":"story","value":"WorkSpaces Creation and Editing Functionality"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"workspaces_test"},{"name":"subSuite","value":"TestWorkspaces"},{"name":"host","value":"fv-az1758-889"},{"name":"thread","value":"2046-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.workspaces_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":0,"unknown":0,"total":4},"items":[{"uid":"f3ead57491610844","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/703//#testresult/f3ead57491610844","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751507865873,"stop":1751507865873,"duration":0}},{"uid":"34298e0ad450f1c5","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/702//#testresult/34298e0ad450f1c5","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751421441867,"stop":1751421441867,"duration":0}},{"uid":"bcb99551db8d097d","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/701//#testresult/bcb99551db8d097d","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1751335727515,"stop":1751335727515,"duration":0}}]},"tags":[]},"source":"9079123045843dd9.json","parameterValues":[]}